src/Findologic/Response/Xml21/Filter/Filter.php

```diff
@@ -108,14 +108,14 @@ private static function handleRangeSliderFilter(ApiRangeSliderFilter $filter): R
 
         if ($filter->getAttributes()->getTotalRange()) {
             $customFilter->setTotalRange([
-                self::FILTER_RANGE_MIN => $filter->getAttributes()->getTotalRange()->getMin(),
+                self::FILTER_RANGE_MIN => round($filter->getAttributes()->getTotalRange()->getMin(), 2),
```

Comment:
Shouldn't we also round the max value?

Reply:
From the my first investigation, we have a problem exclusively with zero decimals... Give me one more try to test again big values. If it is true, and we have only issue with zero values, under one, maybe we can only need to hande min values. Or how you say :)

Reply:
I think it would generally make sense, just to be consistent here with the values.